projects/plugins/crm/src/entities/class-quote.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * Quote Entity.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Entities;
+
+/**
+ * Quote class.
+ *
+ * @since $$next-version$$
+ */
+class Quote {
+
+	/**
+	 * The DB ID of the object.
+	 *
+	 * @var int
+	 */
+	public $id = -1;
+
+	/**
+	 * The owner of the object.
+	 *
+	 * @var int
+	 */
+	public $owner = -1;
+
+	/**
+	 * The quote meta.
+	 *
+	 * @var array
+	 */
+	public $meta = array();
+
+	/**
+	 * The quote tags.
+	 *
+	 * @var array
+	 */
+	public $tags = array();
+
+	/**
+	 * The quote files.
+	 *
+	 * @var array
+	 */
+	public $files = array();
+
+	/**
+	 * The quote notes.
+	 *
+	 * @var array
+	 */
+	public $notes = array();
+
+	/**
+	 * The ID override of the quote.
+	 *
+	 * @var string
+	 */
+	public $id_override = '';
+
+	/**
+	 * The hash of the quote.
+	 *
+	 * @var string
+	 */
+	public $hash = '';
+
+	/**
+	 * The quote title.
+	 *
+	 * @var string
+	 */
+	public $title = '';
+
+	/**
+	 * The date property of the quote.
+	 *
+	 * @var int
+	 */
+	public $date = -1;
+
+	/**
+	 * The value of the quote.
+	 *
+	 * @var float
+	 */
+	public $value = 0.0;
+
+	/**
+	 * The currency of the quote.
+	 *
+	 * @var string ( see zeroBS_buildObjArr in ZeroBSCRM.Dal3.Helpers - currently building curr as str)
+	 */
+	public $currency = '';
+
+	/**
+	 * The content property of the quote.
+	 *
+	 * @var string
+	 */
+	public $content = '';
+
+	/**
+	 * The quote template property.
+	 *
+	 * @var string
+	 */
+	public $template = '';
+
+	/**
+	 * The send attachments property of the quote.
+	 *
+	 * @var int
+	 */
+	public $send_attachments = -1;
```

Comment:
`send_attachments` is a boolean

Reply:
Oh yes, I mistakenly looked at the DB entry instead - https://github.com/Automattic/jetpack/blob/e156ed85552fed62107978a7f41197640c7aa9b9/projects/plugins/crm/includes/ZeroBSCRM.Database.php#L776 and misinterpreted tinyint to be an int. Will fix in another commit, thanks!